src/components/Nav.js

```diff
@@ -18,14 +20,54 @@ const Nav = () => {
       alignSelf="center"
       gap="medium"
     >
-      <RoutedAnchor
-        path="/"
-        icon={<GrommetIcon size="large" />}
-        label={size !== 'small' && <Text size="xlarge">grommet</Text>}
-      />
+      <Box
+        direction="column"
+        justify="between"
+        align="start"
+        alignSelf="center"
+        width="medium"
+      >
+        <RoutedAnchor
+          path="/"
+          icon={<GrommetIcon size="large" />}
+          label={size !== 'small' && <Text size="xlarge">grommet</Text>}
+        />
+        {themeName === 'grommet' && <Text size="small">for open source</Text>}
+        {themeName === 'hpe' && <Text size="small">for HPE</Text>}
+      </Box>
+
       <Box direction="row" gap="small">
         {!searchOpen && (
           <Box direction="row" gap="small">
+            <Button
+              plain
+              onClick={() => {
+                if (themeName === 'hpe') {
+                  setThemeName('grommet');
+                } else {
+                  setThemeName('hpe');
+                }
+              }}
+            >
+              {({ hover }) => (
+                <Box
+                  pad={{ vertical: 'small', horizontal: 'small' }}
+                  round="xlarge"
+                  background={hover ? 'active' : undefined}
+                  direction="row"
+                  gap="xsmall"
+                >
+                  <Text>
+                    {themeName === 'hpe'
+                      ? 'use for open source'
```

Comment:
Proposing we simplify the logic to be:

`Use for ${themeName === 'hpe' ? HPE : open source}`